### ISSUE TITLE
cannot use on 2 computers with database sync

### ISSUE THREAD

**Issue by andreasalberta** [pfn/passifox#608]
it tells me duplicate UUID and then breaks the relationship with the other computer

**Comment by sinni800** [comment]
Yes, that's exactly it. PasslFox will try to save the sync password to the Keepass db... So keepass has to be open for sync to work. Otherwise sync will break all the time!

I have two proposed solutions:

1) Keep KeePass required but have an option where different entries can be targeted by client with some option inside the entry for example. This would be in KeePassHTTP

2) The plugin gets KPH: strings if set. Make it default to yes and give the browser plugin a way to discriminate the one to choose... I wouldn't know what kind of value one would use to choose though

3) Give the plugin an exception list to match against ( add chrome://FirefoxAccounts by default)
